source/install-from-source.txt

```diff
@@ -267,10 +266,10 @@ messages and information. Ensure that configuration succeeds before proceeding.
   results. This should be set to the same value as the version of the source
   driver that was downloaded in :ref:`get-src`.
 
-  The ``ENABLE_EXTRA_ALIGNMENT`` and ``ENABLE_AUTOMATIC_INIT_AND_CLEANUP`` are part
-  of ``mongo-c-driver``, and correspond to deprecated features that are only
+  The ``ENABLE_EXTRA_ALIGNMENT`` is part
+  of ``mongo-c-driver``, and corresponds to a deprecated feature that is only
```

Comment:
S: clarify what ENABLE_EXTRA_ALIGNMENT is; I'd also remove the comma before the dependent clause
```suggestion
  The ``ENABLE_EXTRA_ALIGNMENT`` argument is part
  of ``mongo-c-driver`` and corresponds to a deprecated feature that is only
```